cfme/js.py

```diff
@@ -21,7 +24,9 @@
         && isHidden(document.getElementById("lightbox_div")),
     document: document.readyState,
     autofocus: (typeof checkMiqQE === "undefined") ? 0 : checkMiqQE('autofocus'),
-    miqQE: (typeof checkAllMiqQE === "undefined") ? 0 : checkAllMiqQE()
+    miqQE: (typeof checkAllMiqQE === "undefined") ? 0 : checkAllMiqQE(),
+    miqProcessing: ManageIQ.observe.processing || ManageIQ.observe.queue.length > 0,
```

Comment:
this check should also be guarded as it is not guaranteed that the ManageIQ is defined. Eg. when we see an error, then it would fail without letting us interact with it.